ui/imports/shared/popups/SettingsDirtyToastMessage.qml

```diff
@@ -33,47 +41,20 @@ Rectangle {
     opacity: active ? 1 : 0
     color: Theme.palette.statusToastMessage.backgroundColor
     radius: 8
-    border.color: Theme.palette.dangerColor2
+    border.color: type === SettingsDirtyToastMessage.Type.Danger ? Theme.palette.dangerColor2 : Theme.palette.primaryColor2
     border.width: 2
+
     layer.enabled: true
     layer.effect: DropShadow {
         verticalOffset: 3
         radius: 8
         samples: 15
         fast: true
         cached: true
-        color: Theme.palette.dangerColor2
+        color: root.border.color
         spread: 0.1
     }
 
-    onActiveChanged: {
-        if (!active || !flickable)
-            return;
-
-        const item = Global.applicationWindow.activeFocusItem;
-        const h1 = this.height;
-        const y1 = this.mapToGlobal(0, 0).y;
-        const h2 = item.height;
-        const y2 = item.mapToGlobal(0, 0).y;
-        const margin = 20;
-        const offset = h2 - (y1 - y2);
-
-        if (offset <= 0 || flickable.contentHeight <= 0)
-            return;
-
-        toastFlickAnimation.from = flickable.contentY;
-        toastFlickAnimation.to = flickable.contentY + offset + margin;
-        toastFlickAnimation.start()
-    }
-
-    NumberAnimation {
-        id: toastFlickAnimation
-        target: root.flickable
-        property: "contentY"
-        duration: 150
-        easing.type: Easing.InOutQuad
-    }
-
```

Comment:
Same question here, why?
This was related to the placeholder. When the toast message appears, in case it overlaps the content, it would scroll correspondingly so that you would still see your input field.

Reply:
Exactly, this would always break pages like the one in this PR with long listviews where you want to perform some drag'n'drop operations; as soon as the page gets "dirty", this would push the contents up to make some space which in turn breaks the DND :/

Reply:
TLDR; restored the old placeholder and flickable auto-scroll behavior but made those conditional based on the `ignoreDirty` flag. This way both bubbles work fine